histomicstk/segmentation/label/trace_boundary.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import ctypes
-from .isbf import isbf
+from .trace_object_boundary import trace_object_boundary
 
 
 def trace_boundary(Mask, Connectivity=4, XStart=None, YStart=None,
```

Comment:
Make trace_boundary work on a label mask (with mutliple objects) instead of a binary mask (with one object).

Currently i have to do the following for this:

```python
def get_label_contours(imLabel):
    
    label_id_list = np.unique(imLabel.flatten())

    obj_contours = {}

    for i in range(len(label_id_list)):

        label_id = label_id_list[i]
        
        # create binary mask for current label object
        # BUG: had to pad to make it work on objects at image boundary
        imMask = np.pad(imLabel == label_id, (1, 1), mode='constant')

        # get object boundary coords as an N X 2 array
        X, Y = trace_boundary(imMask)       

        coords = np.vstack((X, Y)).T - 1

        # add object contour to dictionary
        obj_contours[label_id] = coords  
        
    return obj_contours
```

It would be much faster to have that loop over object in cython or c++ file.